modules/ROOT/pages/configure-partner.adoc

```diff
@@ -21,18 +21,22 @@ To create a new partner, you need the following information:
 [[create-partner]]
 == Create a Partner
 
+=== Prerequisite
 Before you create your first trading partner, ensure you configure your xref:setup-payload-storage-API.adoc[payload storage API].
 
-. From within your Anypoint Partner Manager environment, select *Partners* from the left navigation.
-. Click *New partner*.
-. In *Name*, enter a name that easily identifies your partner.
-. Select an identifier from the *Type* list:
+=== Steps
+. In Partner Manager, select *Partners* in the left navigation.
```

Comment:
From the left navigation menu in Partner Manager, select *Partners* > *New partner*?

Reply:
A new page appears when you select **Partners**, so I think I will leave this construction as is.